openlibrary/plugins/upstream/borrow.py

```diff
@@ -783,16 +793,21 @@ def user_can_borrow_edition(user, edition, resource_type):
     waitlist_size = realtime_availability['num_waitlist']
 
     if waitlist_size:
-        # There some people are already waiting for the book,
-        # it can't be borrowed unless the user is the first in the waiting list.
-        waiting_loan = user.get_waiting_loan_for(edition)
-        my_turn_to_borrow = (waiting_loan and waiting_loan['status'] == 'available'
-                             and waiting_loan['position'] == 1)
-        return my_turn_to_borrow
+        return is_users_turn_to_borrow(user, edition)
```

Comment:
BLOCKER: This calling name doesn't match the method name `is_turn_to_borrow`. Was this fix tested on staging?? We should really make sure it works there first.

Reply:
resolved